### DISCUSSION TITLE
Metasploit style resource command

### DISCUSSION THREAD

**Discussion by kernelzeroday** [BishopFox/sliver#555] [Ideas]
It would be nice to be able to run a series of sliver commands from file similar to a metasploit script.rc file.

**Comment by moloch--** [comment]
Not really at this time, however you can use [`sliver-py`](https://github.com/moloch--/sliver-py) to automate any functionality within Sliver. There are also "persistent" flags on several commands (such as c2 listeners) that will, for example, automatically restart listener jobs when you start the server, and `reactions` more or less work like MSF `.rc` files however they only execute in reaction to an event.
